rules/windows/defense_evasion_masquerading_communication_apps.toml

```diff
@@ -35,40 +37,40 @@ process where host.os.type == "windows" and
   (
     /* Slack */
     (process.name : "slack.exe" and not
-      (process.code_signature.subject_name in (
+      (process.code_signature.subject_name : (
```

Comment:
FYI for the reviewers, changing this to be case insensitive as S1 populates this field with all uppercase